samples/matter/thermostat/src/temperature_manager.cpp

```diff
@@ -38,12 +40,14 @@ uint8_t ReturnRemainderValue(int16_t Value)
 
 CHIP_ERROR TemperatureManager::Init()
 {
+	Status status;
+
 	PlatformMgr().LockChipStack();
-	Thermostat::Attributes::LocalTemperature::Get(kThermostatEndpoint, mLocalTempCelsius);
-	Thermostat::Attributes::OutdoorTemperature::Get(kThermostatEndpoint, mOutdoorTempCelsius);
-	Thermostat::Attributes::OccupiedCoolingSetpoint::Get(kThermostatEndpoint, &mCoolingCelsiusSetPoint);
-	Thermostat::Attributes::OccupiedHeatingSetpoint::Get(kThermostatEndpoint, &mHeatingCelsiusSetPoint);
-	Thermostat::Attributes::SystemMode::Get(kThermostatEndpoint, &mThermMode);
+	VerifyOrReturnError(Status::Success == LocalTemperature::Get(kThermostatEndpoint, mLocalTempCelsius), CHIP_IM_GLOBAL_STATUS(Failure));
```

Comment:
`UnlockChipStack` must be called on early return. It's safest to use `DeviceLayer::StackLock` for this reason.